src/context/fetch.ts

```diff
@@ -2,7 +2,10 @@ import { Headers } from 'headers-utils'
 import { MockedRequest } from '../handlers/RequestHandler'
 import { isNodeProcess } from '../utils/internal/isNodeProcess'
 
-const useFetch = isNodeProcess() ? require('node-fetch') : window.fetch
+const useFetch: (
+  input: RequestInfo,
+  init?: RequestInit,
+) => Promise<Response> = isNodeProcess() ? require('node-fetch') : window.fetch
```

Comment:
This cannot be injected at build time because `ctx.fetch` is _shared_ between the browser and NodeJS bundles.